governator-core/src/main/java/com/netflix/governator/InjectorBuilder.java

```diff
@@ -62,6 +63,23 @@ public static InjectorBuilder fromModules(List<Module> modules) {
         return new InjectorBuilder(Modules.combine(modules));
     }
     
+    public static <T> InjectorBuilder bindInstance(final Object o) {
```

Comment:
The InjectorBuilder currently deals with modules as opposed to introducing a new binding API.  I think this needs a lot more discussion before we expand the scope of the InjectorBuilder.   Let's do this as a separate pull request.